packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1591,6 +1674,16 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
     if (widget.selectionEnabled && pasteEnabled && widget.selectionControls?.canPaste(this) == true) {
       _clipboardStatus?.update();
     }
+
+    widget.controller._setInputFormatters(
+      widget.inputFormatters ?? const <TextInputFormatter>[]
```

Comment:
Nit: Comma at the end of this line.

Reply:
Was there something about inputFormatters not being allowed to be empty?  Or I'm thinking of `formatters`.

Reply:
`EditableText.inputFormatters`'s type is `List<TextInputFormatters>?`.